packages/android-edge-to-edge-support/android/src/main/java/io/capawesome/capacitorjs/plugins/androidedgetoedgesupport/EdgeToEdgeConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.capawesome.capacitorjs.plugins.androidedgetoedgesupport;
+
+import android.graphics.Color;
+import com.getcapacitor.PluginConfig;
+
+public class EdgeToEdgeConfig {
+
+    private int backgroundColor = Color.WHITE;
+
+    public EdgeToEdgeConfig(PluginConfig config) {
+        String backgroundColor = config.getString("backgroundColor");
+        if (backgroundColor != null) {
+            this.backgroundColor = Color.parseColor(backgroundColor);
+        }
```

Comment:
Please load the config like that in the `EdgeToEdgePlugin.java` file: https://github.com/capawesome-team/capacitor-plugins/blob/678bf4e4a8caefd150261e37645872c0015439c3/packages/live-update/android/src/main/java/io/capawesome/capacitorjs/plugins/liveupdate/LiveUpdatePlugin.java#L59